### PR TITLE
Fix a bug in kfctl.sh when just creating a K8s app and not GCP.

### PR DESCRIPTION
**Pull request by jlewi**
* We need to set KUBEFLOW_KS_DIR in env.sh for all platforms.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1357)

### CODE REVIEW

File: scripts/kfctl.sh [hunk 1]
```diff
@@ -28,6 +28,7 @@ createEnv() {
 	DEFAULT_KUBEFLOW_REPO="$( cd "${DIR}/.." >/dev/null && pwd )"
     echo KUBEFLOW_REPO=${KUBEFLOW_REPO:-"${DEFAULT_KUBEFLOW_REPO}"} >> ${ENV_FILE}
     echo KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"} >> ${ENV_FILE}
```
Comment (activatedgeek) [inline scripts/kfctl.sh]: There seems to be an indentation mismatch here.

**Comment by jlewi** [comment]
Thanks @activatedgeek; fixed the indent PTAL.

File: scripts/kfctl.sh [hunk 2]
```diff
@@ -26,8 +26,9 @@ createEnv() {
 	# a file for consistency across runs.
 	echo PLATFORM=${PLATFORM} >> ${ENV_FILE}
 	DEFAULT_KUBEFLOW_REPO="$( cd "${DIR}/.." >/dev/null && pwd )"
-    echo KUBEFLOW_REPO=${KUBEFLOW_REPO:-"${DEFAULT_KUBEFLOW_REPO}"} >> ${ENV_FILE}
-    echo KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"} >> ${ENV_FILE}
+	echo KUBEFLOW_REPO=${KUBEFLOW_REPO:-"${DEFAULT_KUBEFLOW_REPO}"} >> ${ENV_FILE}
+	echo KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"} >> ${ENV_FILE}
+	echo KUBEFLOW_KS_DIR=${KUBEFLOW_KS_DIR:-"`pwd`/ks_app"} >> ${ENV_FILE}
```
[the remark this answers is not in the document]
Comment (activatedgeek) [reply scripts/kfctl.sh]: This (line 31) one slipped through.

**Pull request merged**